draft-ietf-rtcweb-jsep.xml

```diff
@@ -1885,15 +1885,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           session-level attributes. The process here is identical to that
           indicated in the Initial Offers section above.</t>
 
-	  <t>The next step is to generate lip sync groups as defined
-	  in <xref target="RFC5888"/>, Section 7.  For each
-	  MediaStream with more than one MediaStreamTrack, a group
-	  of type "LS" MUST be added that contains the mid values for
-	  each MediaStreamTrack in that MediaStream. In some cases
-	  this may result in adding a mid to a given LS group that
-	  was not in that LS group in the associated offer. Although
-	  this is not allowed by <xref target="RFC5888"/>, it is
-	  allowed when implementing this specification. </t>
+	  <t>The next step is to generate session-level lip sync
+	  groups as defined in <xref target="RFC5888"/>, Section
+	  7. All Tracks in the same MediaStream need to be placed in
```

Comment:
Sure, I'm happy to post -12 if we have the changes in time :)

-Ekr


On Sun, Jul 5, 2015 at 8:13 PM, Justin Uberti <notifications@github.com>
wrote:

> In draft-ietf-rtcweb-jsep.xml
> <https://github.com/rtcweb-wg/jsep/pull/167#discussion_r33903866>:
>
> > @@ -1885,15 +1885,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
> >            session-level attributes. The process here is identical to that
> >            indicated in the Initial Offers section above.</t>
> >
> > -	  <t>The next step is to generate lip sync groups as defined
> > -	  in <xref target="RFC5888"/>, Section 7.  For each
> > -	  MediaStream with more than one MediaStreamTrack, a group
> > -	  of type "LS" MUST be added that contains the mid values for
> > -	  each MediaStreamTrack in that MediaStream. In some cases
> > -	  this may result in adding a mid to a given LS group that
> > -	  was not in that LS group in the associated offer. Although
> > -	  this is not allowed by <xref target="RFC5888"/>, it is
> > -	  allowed when implementing this specification. </t>
> > +	  <t>The next step is to generate session-level lip sync
> > +	  groups as defined in <xref target="RFC5888"/>, Section
> > +	  7. All Tracks in the same MediaStream need to be placed in
>
> I have limited access, but let me try to type something up here on Monday.
> I see you posted a new version, but perhaps we could still close out this
> PR in the next day or two.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtcweb-wg/jsep/pull/167/files#r33903866>.
>